### ISSUE TITLE
False-negative on Component properties

### ISSUE THREAD

**Issue by ai** [astralhpi/svelte-fast-check#5]
I have `[<Stack>](https://github.com/hplush/slowreader/blob/main/web/ui/stack.svelte)` component with

```svelte
<script lang="ts">
  let {
    gap = 'm'
  }: {
    gap?: 'l' | 'm' | 's' | 'xl' | 'xs' | 'xxl'
  } = $props()
</script>
```

If I make a mistake in other component:

```svelte
<Stack ga="m">
```

`svelte-check` will show an error:

```
Error: Object literal may only specify known properties, and '"ga"' does not exist in type '$$ComponentProps'. (ts)
<ThinPage title={[$t.pageTitle, $settingsMessages.commonTitle]}>
  <Stack ga="xl">
    <Title>Slow Reader</Title>
```

But `svelte-fast-check` will show nothing:

```
svelte-fast-check
🔍 svelte-fast-check: Starting type check...

Found 108 .svelte files
Converted 108 files successfully
────────────────────────────────────────────────────────────
✓ No problems found
Completed in 780ms
   (typeCheck: 593ms, svelteWarnings: 276ms)
```

(Other mistakes in `.svelte` files will be shown by `svelte-fast-check`)
